staging/src/k8s.io/apiserver/pkg/storage/testing/store_tests.go

```diff
@@ -129,33 +130,40 @@ func RunTestGet(ctx context.Context, t *testing.T, store storage.Interface) {
 		ignoreNotFound    bool
 		expectNotFoundErr bool
 		expectRVTooLarge  bool
-		expectedOut       *example.Pod
```

Comment:
What was forcing this change?

Reply:
The change to a slice is described below - the "NotOlderThan" semantic is making some tests to have multiple potentially correct results.

The change to interface{} was to allow passing a generic thing to the newly created `ExpectNoDiffWithOne` function.
I reverted that and added a helper function for conversion instead.